### DISCUSSION TITLE
unbale to import mp3 in next js

### DISCUSSION THREAD

**Discussion by SahasSaurav** [vercel/next.js#20639] [Help]
even tried to configure next.config.js as list in this article
https://github.com/vercel/next.js/discussions/12810

![Screenshot (94)](https://user-images.githubusercontent.com/60291223/103405671-da46cb80-4b7d-11eb-917c-a7aa219e9b53.png)

the link for file
https://github.com/SahasSaurav/next_pomodoro/blob/main/src/components/PomodoroClock.tsx

setting from  this discussion https://github.com/vercel/next.js/discussions/12810
implemented it can use audio with useSound hook but can't deploy it to vercel

**Comment by jamesmosier** [comment]
Have you tried to move `Timer_Beep.mp3` to your `public` folder and then use it like this:

```js
const [beepPlay,{stop}]=useSound('/Timer_Beep.mp3');
```

You could then do the same for the other mp3 files you have.
